docs/topics/funding.md

```diff
@@ -306,22 +316,15 @@ Our professional and premium plans also include **priority support**. At any tim
 
 *Billing is monthly and you can cancel at any time.*
 
-Once you've signed up I'll contact you via email and arrange your ad placements on the site.
+Once you've signed, we will contact you via email and arrange your ad placements on the site.
 
 For further enquires please contact <a href=mailto:funding@django-rest-framework.org>funding@django-rest-framework.org</a>.
 
 ---
 
 ## Accountability
 
-In order to ensure that I can be fully focused on trying to secure a sustainable
-& well-funded open source business I will be leaving my current role at [DabApps](http://www.dabapps.com)
-at the end of May 2016.
-
-I have formed a UK limited company, [Encode](http://www.encode.io), which will
-act as the business entity behind REST framework. I will be issuing monthly reports
-from Encode on progress both towards the [Mozilla grant](mozilla-grant.md), and for development time
-funded via the REST framework paid plans.
+In an effort to keep the project as transparent as possible, we are releasing [monthly progress reports](http://www.encode.io/reports/february-2017) and regularly include financial reports and cost breakdowns.
```

Comment:
👍  - We'll need to remember to update this link once we've got an index to point at.